.github/workflows/ci.yaml

```diff
@@ -1,44 +1,67 @@
-name: ci
+name: CI
 
-on:
-  push:
-    branches: [ "main" ]
-  pull_request:
-    branches: [ "main" ]
+on: ["push", "pull_request"]
```

Comment:
this will run on every commit then right? to validate the dev branches before opening the PR

Reply:
yes, exactly